pom.xml

```diff
@@ -237,20 +238,17 @@
                             <goal>jar</goal>
                         </goals>
                         <configuration>
-                            <source>8</source>
                         </configuration>
                     </execution>
                 </executions>
             </plugin>
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-compiler-plugin</artifactId>
-                <version>3.6.1</version>
+                <version>3.8.1</version>
```

Comment:
Why not the latest 3.10.1 version?

Reply:
This PR is old, that's why it's an old version

Reply:
Due to many of our users are still using Java 8, we cannot migrate to Java 11  in the current form. We are open to all options to support both Java 8 and Java 11.